yarn.lock

```diff
@@ -4561,7 +4589,7 @@ postcss-import@^15.1.0:
     read-cache "^1.0.0"
     resolve "^1.1.7"
 
-postcss-js@^4.0.1:
+postcss-js@^4, postcss-js@^4.0.1:
```

Comment:
Are the changes made here  necessary ???

Reply:
The above were auto-generated

Reply:
Same here